composer.json

```diff
@@ -30,7 +30,8 @@
         "guzzle/guzzle": "~3.7",
         "ornicar/akismet-bundle": "*@dev",
         "ornicar/gravatar-bundle": "dev-master@dev",
-        "antimattr/google-bundle": "dev-master@dev"
+        "antimattr/google-bundle": "dev-master@dev",
```

Comment:
After looking at the package name it's a different package then what is used for youtube. Maybe it's out of scope for this PR?